packages/lesswrong/components/Layout.tsx

```diff
@@ -345,6 +345,14 @@ const Layout = ({currentUser, children, classes}: {
                 sidebarHidden={hideNavigationSidebar}
                 toggleStandaloneNavigation={toggleStandaloneNavigation}
                 stayAtTop={Boolean(currentRoute?.fullscreen || currentRoute?.staticHeader)}
+                currentEvent={
+                  isEAForum
+                    ? {
+                      name: "Career week",
+                      link: "#",
```

Comment:
I think this should link to the post.

I would probably made this a db setting.

Reply:
Whoops - forgot it was a placeholder!